Top level string test is failing

Interestingly, the top level string test appears to have been failing ever since it was introduced in d95581062de5fbdf5377b29d14160045b9eff1a0 . It's failing today in `master`, and if you check out that commit, it's failing there too. No idea why... It should probably be fixed. I'm not sure if the expected value is wrong or the output is wrong. It can be verified by applying the ops to a document and seeing what works correctly.

```
  21 passing (24ms)
  1 failing

  1) Jsondiff
       Operations
         String Mutation (si + sd)
           Top level string:

      AssertionError: expected [ Array(2) ] to deeply equal [ Array(2) ]
      + expected - actual

       [
         {
      -    "p": [
      -      0
      -    ]
      +    "p": []
           "sd": "one"
         }
         {
      -    "p": [
      -      0
      -    ]
      +    "p": []
           "si": "two"
         }
       ]
```

--- a/test/index.test.js
+++ b/test/index.test.js
@@ -5,7 +5,7 @@ let assert = require("assert");
 // Library we're testing
 let jsondiff = require("../index.js");
 // json0 transform to work out if the right transform is being created
-let json0 = require("ot-json0/lib/json0");
+let json0 = require("ot-json0");
 // Assertion expectations
 let expect = require("chai").expect;
 // Library for computing differences between strings
@@ -244,8 +244,8 @@ describe("Jsondiff", function() {
           start: "one",
           end: "two",
           expectedCommand: [
-            { sd: "one", p: [] },
-            { si: "two", p: [] }
+            { sd: "one", p: [0] },
+            { si: "two", p: [0] }
           ]
         },
         {
@@ -305,6 +305,10 @@ describe("Jsondiff", function() {
         it(test.name, function() {
           let output = jsondiff(test.start, test.end, diffMatchPatch);
           expect(output).to.deep.equal(test.expectedCommand);
+
+          // Test actual application of the expected command.
+          let appliedEnd = json0.type.apply(test.start, test.expectedCommand);
+          expect(appliedEnd).to.deep.equal(test.end);
         });
       });
     });

